### PR TITLE
[IMP] im_livechat: remove PublicLivechatMessageChatbot

### PR DESCRIPTION
**Pull request by alexkuhn**
Task-2898364

### CODE REVIEW

File: addons/im_livechat/static/src/legacy/models/public_livechat_message.js [hunk 1]
```diff
@@ -109,6 +115,30 @@ const PublicLivechatMessage = Class.extend({
...
+        return this._chatbotStepAnswers;
+    },
+    /**
+     * Get chat bot script answer ID
+     *
+     * @return {string}
+     */
+    getChatbotStepAnswerId() {
+        return this._chatbotStepAnswerId;
+    },
```
Comment (oomsveta) [inline addons/im_livechat/static/src/legacy/models/public_livechat_message.js:141]: useless comments

File: addons/im_livechat/static/src/legacy/models/public_livechat_message.js [hunk 2]
```diff
@@ -408,6 +438,12 @@ const PublicLivechatMessage = Class.extend({
             return _.contains(attachmentIDs, attachment.id);
         });
     },
+    /**
+     * @override
+     */
+    setChatbotStepAnswerId(chatbotStepAnswerId) {
```
Comment (oomsveta) [inline addons/im_livechat/static/src/legacy/models/public_livechat_message.js:444]: doesn't look like an override

**Comment by alexkuhn** [comment]
Changes done

**Review by oomsveta — APPROVED** [review]

**Comment by alexkuhn** [comment]
Merged in grouped PR: https://github.com/odoo/odoo/pull/96447

**Pull request closed without merging**